lib/remarkMdxLayout.mjs

```diff
@@ -30,14 +30,15 @@ export const remarkMdxLayout = () => {
   
           const outline = ${Serializer.run(outline)}
   
-          export const getStaticProps = async () => {
+          export const getStaticProps = async (params) => {
```

Comment:
Should probably be `context` instead of `params`. Also, this is just in case `mdxGetStaticProps` might need the context (which doesn't seem to be the case in your example), right?

Reply:
yeah but I added it in case some another mdxGetStaticProps will need it in the future ;)